File: java/contador.java
import java.util.Scanner;

public class contador {
	public static void main (String[] args) {
		Scanner v = new Scanner(System.in);
		System.out.println("Qual o valor máximo");
		int n = v.nextInt();
		int res = 0;
		for (int i = 0;i<n;i++) {
			System.out.println("Insira os valores");
			int val = v.nextInt();
			if(val % 1 == 0) {
				res += 1;
			}
			System.out.println("Há " + res + " valores inteiros");
		}
	}
}

File: javascript/contador.js

var tam = parseInt(prompt("Qual o número máximo?"))
var result = 0
for(var i  = 0; i<tam; i++){
    var valores = prompt("Alimente o vetor")
    if(valores % 1 === 0){
        result += 1
    }
}
document.write("Ha " + " valores inteiros")
